dags/hivemind_etl_helpers/src/db/mediawiki/mediawiki_reader.py

```diff
@@ -0,0 +1,49 @@
+"""Simple reader that reads mediawiki."""
+
+from typing import Any, List, Optional
+
+from llama_index.legacy.readers.base import BasePydanticReader
+from llama_index.legacy.schema import Document
+
+
+class MediaWikiReader(BasePydanticReader):
+    """WikiMedia reader.
+
+    Reads a page.
+
+    """
+
+    is_remote: bool = True
+
+    def __init__(self, api_url: Optional[str] = None) -> None:
+        """Initialize with parameters."""
```

Comment:
Please consider importing wikipedia at the header of this file so could be used everywhere.